assets/main.js

```diff
@@ -0,0 +1,134 @@
+class Book {
+  constructor(author, title, pages, read = 'Unread') {
+    this.author = author;
+    this.title = title;
+    this.pages = pages;
+    this.read = read;
+  }
+}
+
+const getBooksFromLocal = () => JSON.parse(localStorage.getItem('books')) || [];
+
+function addBookToLocal(book) {
+  const books = getBooksFromLocal();
+  books.push(book);
+
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function findBook(title, author, pages) {
+  const books = getBooksFromLocal();
```

Comment:
- [ ] This function always returns -1. When you delete a book, it's deleted from the view, but it is still persisted in the localStorage on the next refresh. Let's fix this. See my next comment.